apps/teritori-nft/app.json

```diff
@@ -0,0 +1,11 @@
+{
+  "appCategory": "NFT",
+  "appName": "Teritori - NFT",
```

Comment:
the formatting is like this, because there are many other apps by teritori to add